### PR TITLE
TMA-484: Fix getting latest master version (#1258)

### PR DESCRIPTION
**Pull request by Hahihula**
Test succeded locally

### CODE REVIEW

File: lib/gooddata/lcm/actions/collect_segment_clients.rb [hunk 1]
```diff
@@ -49,16 +47,11 @@ def call(params)
...
-
-            path = File.expand_path('../../data/select_from_lcm_release.sql.erb', __FILE__)
-            query = GoodData::Helpers::ErbHelper.template_file(path, replacements)
-
-            res = params.ads_client.execute_select(query)
-            latest_master_id = res.max_by { |row| row[:version] }[:master_project_id]
+            latest_master_id = GoodData::LCM2::Helpers.latest_master_project(
+              params.release_table_name,
+              params.ads_client,
+              segment.segment_id
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/collect_segment_clients.rb]: Calls 'segment.segment_id' 2 times (DuplicateMethodCall)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+module GoodData
+  module LCM2
+    class Helpers
+      DEFAULT_TABLE_NAME = 'LCM_RELEASE'
+
+      class << self
+        def latest_master_project(release_table_name, ads_client, segment_id)
+          replacements = {
+            table_name: release_table_name || DEFAULT_TABLE_NAME,
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Is controlled by argument 'release_table_name' (ControlParameter)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module LCM2
+    class Helpers
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has no descriptive comment (IrresponsibleModule)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      DEFAULT_TABLE_NAME = 'LCM_RELEASE'
+
+      class << self
+        def latest_master_project(release_table_name, ads_client, segment_id)
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has approx 7 statements (TooManyStatements)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module LCM2
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has the name 'lcm2' (UncommunicativeModuleName)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/411/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1256/ : SUCCESS in 1m 15s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1526/ : SUCCESS in 1m 03s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/110/ : SUCCESS in 44s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1265/ : SUCCESS in 2m 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1535/ : SUCCESS in 3m 08s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1482/ : SUCCESS in 17m 31s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/54/ : FAILURE in 2m 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/111/ : SUCCESS in 20s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1266/ : SUCCESS in 2m 49s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1536/ : SUCCESS in 2m 34s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1483/ : SUCCESS in 15m 19s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/55/ : SUCCESS in 45m 11s

**Pull request merged**